packages/mobile/src/web3/contracts.ts

```diff
@@ -2,25 +2,61 @@ import { newKitFromWeb3 } from '@celo/contractkit'
 import { privateKeyToAddress } from '@celo/utils/src/address'
 import { Platform } from 'react-native'
 import * as net from 'react-native-tcp'
+import { select, take } from 'redux-saga/effects'
+import sleep from 'sleep-promise'
 import { DEFAULT_FORNO_URL } from 'src/config'
 import { IPC_PATH } from 'src/geth/geth'
-import networkConfig from 'src/geth/networkConfig'
+import { store } from 'src/redux/store'
 import Logger from 'src/utils/Logger'
+import { Actions } from 'src/web3/actions'
+import { contractKitReadySelector, fornoSelector } from 'src/web3/selectors'
 import Web3 from 'web3'
 import { provider } from 'web3-core'
 
-// Logging tag
 const tag = 'web3/contracts'
 
-const web3: Web3 = getWeb3()
-let contractKit = newKitFromWeb3(web3)
+export const web3ForUtils: Web3 = new Web3() // Web3 with no provider
```

Comment:
Are there potential downsides to having two web3 instances created at the same time?
Instead of this, I'd recommend creating a `getWeb3` function that proxies to the kit's existing web3 instance

Reply:
Switched to avoid creating an additional web3 instance. 

There are still two instances of contractKit (which each have their own web3 instance) for each mode, but this was suggested by Mariano (https://celo-org.slack.com/archives/CL7BVQPHB/p1581115835017900) as a better way than switching providers